roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -115,7 +108,7 @@ remote_id_attribute = {{ sp.remote_id_attribute }}
 
 {% if keystone.federation.enabled|bool  and keystone.federation.sp.saml.enabled|bool -%}
 
-[saml]
+[saml2]
```

Comment:
Yes, (or it was changed in Mitaka?), but we should be able to set saml2 plugin in the auth plugins and Keystone will automatically know it should be the Mapped class.